### ISSUE TITLE
Screen reader does not read the button name in a popup properly

### ISSUE THREAD

**Issue by sushmita-m** [MicrosoftDocs/msteams-docs#4881]
We have a button in a popup. The screen reader reads the button as `table + button name` as opposed to `button name` 
Attached video for reference:

https://user-images.githubusercontent.com/63236940/146760156-d4ae6970-123e-40a6-8738-988545084b0b.mov

**Comment by Meghana-MSFT** [comment]
We are looking into this issue. We will update you.

**Comment by Meghana-MSFT** [comment]
We are able to repro this. We have raised a bug for the same.

**Comment by Meghana-MSFT** [comment]
We are able to repro this. We have raised a bug for the same.

**Comment by v-anvashist** [comment]
Hi Team,

Any update on the same?

Thanks,
Anjali

**Comment by Meghana-MSFT** [comment]
Currently we do not have any update to share, we will keep you posted if there are any updates.

**Comment by aosolis** [comment]
This is an iframe-based task module and you fully control the HTML displayed inside the page. Please explain why you think this is a Teams issue, as it is not.

It looks like you have your button embedded in a "table" display element, and Chrome is announcing the navigation into the table. Note that Narrator doesn't announce "table" when you navigate backwards to the button (from the "troubleshooting" link) since the user is still in the table.
